index.html

```diff
@@ -11,169 +11,223 @@
     
   </head>
   <body>
-    <div class="maincont">
-
+    <div class="outer-maincont">
       <header class="app-bar">
-        <nav class="lower-bar">
-          <a href="#" class="logoo">Somarib Emmanuel</a> 
-          <div class="menu-icons"><a href="#"><img class="menuimg" src="image/menu.png" alt="menu icon"></a></div>
-        </nav>
+
+          <nav class="lower-bar">
+              <ul><li><a href="#" class="logoo">Somarib Emmanuel</a></li></ul>
+              <ul class="big-menu">
+                  <li><a href="">Portfolio</a></li>
+                  <li><a href="">About</a></li>
+                  <li><a href="">Contact</a></li>
+                  <li><a href=""><img class="" src="image/bg-envlope.png" alt="menu icon"></a></li>
+              </ul>
+              <ul class="menu-icons"><li><a href="#"><img class="menuimg" src="image/menu.png" alt="menu icon"></a></li></ul>
+          </nav>
       </header>
+      <main class="main-cont">
+          <article class="header-section">
+              <div class="header-inner-cont">
+                  <section class="inner-section">
+                      <h1 class="myname">Somarib Emmanuel</h1>
+                      <h2 class="jobtitle">Software Developer</h2>
+                      <p class="abilities">
+                          I can help you build a product , 
+                          feature or website Look through some of my work and experience! 
+                          If you like what you see and have a project you need coded,  
+                          don’t hestiate to contact me.
+                        </p>
```

Comment:
Kindly change the color of this paragraph to match the Figma design 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/187038425-b7de60fb-3a1c-444d-949c-cef66c23f47b.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187038466-ee5b6974-bf40-440d-8ad0-e848b312284e.png)